iocore/net/SSLUtils.cc

```diff
@@ -2152,39 +2154,39 @@ ssl_extract_certificate(const matcher_line *line_info, ssl_user_config &sslMultC
       continue;
     }
 
-    if (strcasecmp(label, SSL_IP_TAG) == 0) {
+    if (SSL_IP_TAG.compare(label) == 0) {
```

Comment:
This isn't the same - `std::string::compare` is case sensitive. If you include "tscpp/util/TextView.h" you get an overload of `strcasecmp` that is specialized for `std::string`, which would not require changes here (i.e. `strcasecmp(label, SSL_IP_TAG)` would work as expected). There is no `strncasecmp` overload because it's not needed.

I realize putting the overload in "tscpp/util/TextView.h" isn't the best option, but I can't put it in "<string_view>".

Reply:
Well, I'll include "tscpp/util/TextView.h" for now.

Reply:
It worked and minimized changes.

> I realize putting the overload in "tscpp/util/TextView.h" isn't the best option, but I can't put it in "<string_view>".

Put it in something like "tscpp/util/StringViewHelper.h"?

Reply:
Meh. We have too many header files already. There's "tscore/ink_string.h" _and_ "tscore/ink_string++.h", those might be choices except they're on my KIWF list.